Gems/PhysX/Code/Source/Collision.cpp

```diff
@@ -46,6 +46,12 @@ namespace PhysX
                 physx::PxPairFlag::eNOTIFY_TOUCH_LOST |
                 physx::PxPairFlag::eNOTIFY_CONTACT_POINTS;
 
+            // resolving contacts between two kinematic objects is invalid
+            if (physx::PxFilterObjectIsKinematic(attributes0) && physx::PxFilterObjectIsKinematic(attributes1))
+            {
+                pairFlags &= ~physx::PxPairFlag::eSOLVE_CONTACT;
```

Comment:
Nice, I had the same change done locally!
`pairFlags.clear(physx::PxPairFlag::eSOLVE_CONTACT);` would be a bit more readable 

Could you also add the same code to `DefaultFilterShaderCCD` too?

Reply:
Cool! Thanks! Will do.

Reply:
Done!